core/templates/pages/library-page/search-selectors/subject-selector.component.html

```diff
@@ -0,0 +1,133 @@
+<div class="filter-container">
+  <mat-card class="filter-box">
+    <mat-card-content>
+      <h3>{{'I18N_LIBRARY_SELECT_CATEGORIES' | translate}}</h3>
+      <hr class="custom-divider">
+    </mat-card-content>
+    <mat-card-content>
+      <div class="checkbox-container" *ngIf="!isMobileViewActive()">
+        <mat-checkbox class="oppia-checkbox"
+          *ngFor="let option of selectionDetails.categories.masterList | slice:0:5"
+          [ngModel]="selectionDetails.categories.selections[option.id]"
+          (ngModelChange)="toggleSelection('categories', option.id)">
+          {{ option.text | translate }}
+        </mat-checkbox>
+      </div>
+      <span class="custom-arrow">&gt;</span><a class="custom-link" (click)="openSelectorModal()">All subjects</a>
+    </mat-card-content>
+  </mat-card>
+</div>
+
+<style>
```

Comment:
Can we put all the CSS rules in one place? Either in the CSS file (```core/templates/pages/library-page/search-selectors/subject-selector.component.css```) or here itself.

Reply:
done